wideq/ac.py

```diff
@@ -7,6 +7,28 @@
 from .core import FailedRequestError
 
 
+class ACJetMode(enum.Enum):
+    """ To turn on the jet mode. This mode
+    puts your AC into highest cooling or
+    dry or heat mode(for a certain amount of time)
+    depending on what you choose
+
+    This mode Overrides following setting:
+    1. Vertical swing is set to @100
+    2. Temperature gets set to 18 after
+    jet mode turns off
+    3. Fan speed is set to HIGH
+    (@AC_MAIN_WIND_STRENGTH_HIGH_W)
+    after jet mode turns off
+    """
+
+    OFF = "@OFF"
+    COOL_JET = "@COOL_JET"
+    HEAT_JET = "@HEAT_JET"
+    DRY_JET = "@DRY_JET_W"
+    HIM_COOL = "@HIMALAYAS_COOL"
```

Comment:
Maybe some simpler names on the Python side would suffice here? Like `OFF`, `COOL`, `HEAT`, `DRY`, and `HIMALAYAS`.

Reply:
done